packages/@react-spectrum/text/src/Heading.tsx

```diff
@@ -37,7 +31,7 @@ function Heading(props: HeadingProps, ref: DOMRef<HTMLHeadingElement>) {
   let HeadingTag = `h${level}` as ElementType;
 
   return (
-    <HeadingTag {...domProps} {...styleProps} ref={domRef}>
+    <HeadingTag {...filterDOMProps(otherProps, {propNames: slotDOMProps})} {...styleProps} ref={domRef}>
```

Comment:
Does this now mean that `aria-current` can be passed as a direct prop in addition to via a slot? why?

Reply:
ahhh, that makes sense, i was trying to figure out what was going on
I'll write a test for that and address it